### PR TITLE
capstone project

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
This project consist of a combination of all the topics I learned, being my first month as a microverse student. I  built a web page for a conference that supports celebration, believing in the value of openness and sharing. In creating a positive change working with varieties of people from different countries. The following were applied here:
-I used semantic HTML tags in this project.
-I used CSS selectors correctly.
-I Used CSS box model.
-I used Flexbox to place elements in the page.
-I Demonstrated the ability to create UIs adaptable to different screen sizes using media queries.
-I used GitHub Pages to deploy web pages.
-I Applied JavaScript best practices and language style guides in this code.
-I Used JavaScript to manipulate DOM elements.
-I Used JavaScript events.
-I used objects to store and access data.
Here is a zoom recording link explaining my portfolio for loom video
https://www.loom.com/share/5d29be8990ab439aab3c22f687579222
## Built With

- HTML for layout 
- CSS for styling
- Linters HTML/CSS and Javascript
-Javascript for language

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            <li class="nav-links">
+              <a href="#partners">Sponsor</a>
+            </li>
+            <li class="nav-links">
+              <a href="about.html">About</a>
+            </li>
+              <li class="nav-links campaign_link">
+                <a href="#contact">Campaign</a>
+              </li>
+          </ul>
```
Comment (mckent05) [inline index.html]: - The links in your navbar are not arranged as they are in the design template. Please try to fix this 🙏 . Please note that replicating the exact design specs gives confidence to recruiters in your ability to replicate design specifications when looking to hire you 😄 
Your project looks like this:
nav project student

Design template:
nav capstone figma

File: index.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+          The theme this 2022 is “CHANGING THE MIND“.
+        </p>
+        <h2 class="date_of_event">
+          2022.11.25(FRI) ~ 26(SAT)
+        </h2>
+        <h3 class="event_address">
+          @Hilton hotel confernce hall yaounde,Cameroon.
+        </h3>
+      </div>
+    </section>
```
Comment (mckent05) [inline index.html]: - Please note that there are some discrepancies between the home page for your site and the home page in the design template. Please try to fix this and try to replicate the design template as closely as possible, and pay keen attention to the fonts, font size, colors, margins e.t.c 🙏  🙏. Replicating exact design specs give confidence to recruiters in your ability to build exact design templates 😄 😄 
Your project (Pay attention to the font size of your title header 😄):
home project student

Design template:
home project figma

File: about.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+      <h3 class="contact_us_text">
+        Please contacts us via email  for any further questions 
+        about diseases, heakth  and the conference 2022.
+      </h3>
+      <h3 class="email_address"><a href="#">
+        emmanuellangufor@gmail.com <span><img class="fordward_arrow" src="images/icons/icons_forward.png" alt="forward arrow"></span>
+        </a>
+      </h3>
+        
+    </section>
```
Comment (mckent05) [inline about.html:107]: Please note that there are some discrepancies between the about page for your site and the home page in the design template. Please try to fix this and try to replicate the design template as closely as possible, and pay keen attention to the fonts, font size, colors, margins e.t.c 🙏 🙏. Replicating exact design specs give confidence to recruiters in your ability to build exact design templates 😄 😄
Your project (Pay attention to the font size of your title header 😄):
about student
Deign Template:
about figma

File: about.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+          <div class="past_stem_image_cover">
+            <h3 class="past_stem_Date">
+              2021
+            </h3>
+
+            <p class="past_stem_text">Changing the mind conference</p>
+
+          </div>
+
+        </div>
```
Comment (mckent05) [inline about.html:156]: - Please note that the past project section(mobile version ) has some deviations from the original design template. The links in the mobile version should be arranged in a vertical column. Please try to fix this.
Your project:
past event student

Design template:
past event figma

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
almost there

## Highlights
- Descriptive commit messages 💯 
- Linters are passing correctly ✔️ ✔️ 
- Your project looks really nice 👍 👍 

## Suggested changes
- Though I see that you have included a link for your loom video in your PR summary 👍  , but I cannot access the link because the given path can only be accessed in your system. Please I'd like to advise that you record your video [online](https://www.loom.com) or upload the video you have done to [loom](https://www.loom.com) Please note that this video is very vital in having your project approved 🙏  
-  I'd advise that you make the background color of your mobile nav transparent or the color of the topbar of the desktop navigation (#272a31) 👍 👍 . Doing this will ensure that your design remains consistent with the desktop version 😄 😄 

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html (hunk 4 above)
Comment (Ngufor-emmanuella) [reply about.html:156]: made all the changes thanks

File: index.html [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+    </header>
+ 
+    <section class="header_section" id="home">
+      
+      <div class="header_section_main_div">
+        
+
+        <div class="header_text_div">
+       <h2 class="stem_by_us"> Living Hapily with Diabetes AFRICAN DIABETES ASSOCIATION 2022!</h2>
+        </div>
```
Comment (Munsa1) [inline index.html]: - [ ] Like the previous reviewer stated your design does not match the given design, kindly add the `text` on top of the page title in the given design which says 'Hello! sharing world'
- [ ] Also change the `h2` to `h1` as this is the main heading for the page and as such, it should be wrapped in an `h1` tag as it is the correct semantic tag.


Given design
image

Your design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 5 above)
Comment (Ngufor-emmanuella) [reply index.html]: Please I have made all changes as requested thanks

**Review by franclobo — APPROVED** [review]
Hi @Ngufor-emmanuella, 

# STATUS: APPROVED 😄 
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Optional suggestion
- [ ] **[OPTIONAL]** Kindly fix the date numbers for a better user interface. 👇🏽
 
image


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**